### PR TITLE
add details for function documentation

### PR DESCRIPTION
**Pull request by jchen9314**

### CODE REVIEW

File: prepackPy/prepackPy.py [hunk 1]
```diff
@@ -12,63 +12,76 @@ def splitter(X, target_index, split_size, seed):
...
+    a proportion of the dataset to include in the test split
 seed: integer
-    the random state that will make code reproducible
+    a random state that will make code reproducible
 
 Returns
 -------
 X_train: numpy array
-    the splitted features for model training
+    a splitted features for model training
```
Comment (apchiodo) [inline prepackPy/prepackPy.py]: maybe just "split features for model training"?

File: prepackPy/prepackPy.py [hunk 2]
```diff
@@ -12,63 +12,76 @@ def splitter(X, target_index, split_size, seed):
...
 X_test: numpy array 
-    the splitted features for model testing
+    a splitted features for model testing
```
Comment (apchiodo) [inline prepackPy/prepackPy.py]: same as above ^^^, same for below vvv

File: prepackPy/prepackPy.py (hunk 1 above)
Comment (jchen9314) [reply prepackPy/prepackPy.py]: OK, I will fix it.

**Comment by apchiodo** [comment]
LGTM after grammar fix.

**Pull request merged**